coderedcms/fields.py

```diff
@@ -11,3 +12,16 @@ def __init__(self, *args, **kwargs):
     def formfield(self, **kwargs):
         kwargs['widget'] = ColorPickerWidget
         return super(ColorField, self).formfield(**kwargs)
+
+
+class ScriptField(models.TextField):
+    def __init__(self, *args, **kwargs):
+        super(ScriptField, self).__init__(*args, **kwargs)
```

Comment:
Since you are not adding any custom logic here, you can skip overriding the `__init__` function. It is essentially doing nothing.

Reply:
done